plasmapy/atomic/particle_class.py

```diff
@@ -540,6 +540,9 @@ def __ne__(self, other) -> bool:
         """
         return not self.__eq__(other)
 
+    def __hash__(self):
+        return hash(self.__repr__())
```

Comment:
You got any more of them cat emojis? I could also use one right now.

To the best of my current and badly limited knowledge, I think this is okay.